### ISSUE TITLE
Display full array values when assertEquals() fails

### ISSUE THREAD

**Issue by benignant** [sebastianbergmann/phpunit#4990]
Is it possible to display full Expected and Actual array values instead of a diff when `assertEquals()` fails? In certain cases, a diff can actually obscure the real problem. In those cases, I would like to have the option to just output the whole array.

Instead of the output documented [here](https://phpunit.readthedocs.io/en/9.5/assertions.html#appendixes-assertions-assertequals-example5), we would get:
```php
# Expected
Array (
    0 => 'a'
    1 => 'b'
    2 => 'c'
)
# Actual
Array (
    0 => 'a'
    1 => 'c'
    2 => 'd'
)
```

**Comment by sebastianbergmann** [comment]
I do not think that this warrants the maintenance cost of a new configuration option.

The information you need is already available: when a comparison fails, the `ComparisonFailure` exception carries both the expected and actual values. You can access them today without any changes to PHPUnit by using an event subscriber that listens to `Test\Failed` events and dumps the full values using `var_export()` or `print_r()`.
